src/Hydra/Serializer/DocumentationNormalizer.php

```diff
@@ -459,7 +459,7 @@ private function getProperty(PropertyMetadata $propertyMetadata, string $propert
     {
         $propertyData = [
             '@id' => $propertyMetadata->getIri() ?? "#$shortName/$propertyName",
-            '@type' => $propertyMetadata->isReadableLink() ? 'rdf:Property' : 'hydra:Link',
+            '@type' => false === $propertyMetadata->isReadableLink() ? 'hydra:Link' : 'rdf:Property',
```

Comment:
Why?

Reply:
Because `readableLink` could be `null`. But not sure what's the correct default / right thing to do here...

Reply:
Oh yeah, we should not have `hydra:Link` when `readableLink` is `null`, so this is correct.